### DISCUSSION TITLE
Who uses TinyMCE?

### DISCUSSION THREAD

**Discussion by androb** [tinymce/tinymce#7323] [Show and tell]
If your company is using TinyMCE in production, please let us know who are you so we can mention you

* Share your company name
* Share your logo (in which case it might be used in the readme/frontpage etc)
* What kind of project? How large? (codebase size or team size)
* If your project is open source or available publicly, please share a link!

**Comment by lrusso** [comment]
![logo](https://user-images.githubusercontent.com/11498901/51078016-90f9ca80-168d-11e9-984b-bd9d17ddb759.png)

http://www.lrusso.com

* What kind of projects LRusso.com develops?

A PHP server provides a desktop interface with file management capabilities, image editing functions for mobile devices and TinyMCE is the document editor. This infrastructure allows to write the required material for press releases of the developed projects. Here are three examples of this:

- Blind Communicator (Free and Open Source App that allows the visually impaired to use a smartphone or tablet) that got great reviews in several newspapers and on the BBC (https://www.youtube.com/watch?v=zboJ-UvUJRU) and CNN (https://www.youtube.com/watch?v=jw5E-W2fir4).

- Mobilty Launcher (Free and Open Source App that allows the mobility impaired to use a smartphone or tablet).

- 3D Object Maker (Free and Open Source 3D designing software for schools and institutions).

Example screenshot of TinyMCE running in the server:

![bf07e07f-118c-45bd-b301-21bbdfbf4680](https://user-images.githubusercontent.com/11498901/51078198-81c84c00-1690-11e9-9be0-59714cd1128c.png)

* GitHub repositories:

https://github.com/lrusso/BlindCommunicator
https://github.com/lrusso/MobilityLauncher
https://github.com/lrusso/3DObjectMaker

**Comment by androb** [comment]
@lrusso that is awesome! Thanks for posting and good luck with the projects. The 3DObjectMaker looked interesting.

**Comment by jipexu** [comment]
![image](https://user-images.githubusercontent.com/328101/51193431-20e78080-18e9-11e9-97a9-7c188e629b85.png)

NPDS CMS use since yearsss ! 

```
 * $RCSfile: tiny_mce.js,v $
 * $Revision: 1.256 $
 * $Date: 2005/06/23 12:04:39 $
```

- open source project
- small team ...

in several place in the cms (user article, reviews, images galerie,  etc)
http://www.npds.org/

 - Currently Tiny 4.8 on our last beta
https://github.com/npds/npds_dune


thanks for your works !

**Comment by alecpl** [comment]
Roundcube is using TinyMCE. https://roundcube.net/

**Comment by Paril** [comment]
![planetminecraft_logo](https://user-images.githubusercontent.com/1224734/51288795-c7a44d80-19cb-11e9-8bdf-f1d69fcc5300.png)

https://www.planetminecraft.com

We've used TinyMCE as our primary WYSIWYG editor for a few years, since 3.x. We're in the process of upgrading to 4.x, and eventually 5.x. We use it for many different purposes.

Large project in terms of reach and end-users, but only a development team of two.

**Comment by Anakowi** [comment]
I first added TinyMCE to my project in 2011. Frustration with Wordpress led me to build my own barebones CMS in 2007. TinyMCE totally lifted the app - so I gained a tiny userbase. In the last few weeks I've been rebuilding the app since PHP4 is now deprecated and causing errorlogs. My users will be rapt when I push out Penpages v2 to them (how slack I am!). Now they can upload images. I'll add a link to the demo site when I've got everything sorted.

**Comment by labby** [comment]
![LEPTON](https://www.opensourcecms.com/wp-content/uploads/LEPTON_CMS_LOGO-300x72.png)

https://lepton-cms.org

Right from the start in 2010 LEPTON cms uses tinymce as standard wysiwyg editor, and it still does. We have included 4.9.2 yet and are now working on integration of tinymce 5 series.

**Comment by cfconsultancy** [comment]
I use TinyMCE in my FLATCMS, demo and preview https://www.flatcms.nl

![2019-01-30_142738](https://user-images.githubusercontent.com/1576475/51984316-5076bb00-249b-11e9-821e-f04792a14345.jpg)

**Comment by lostkeys** [comment]
Hey @cfconsultancy! You are building your admin with bootstrap. We have a bootstrap skin in the works for TinyMCE 5 and it would be cool if you'd like to try it out. If so, get in touch at fredrik.danielsson@tiny.cloud. Cheers!

**Comment by cfconsultancy** [comment]
> Hey @cfconsultancy! You are building your admin with bootstrap. We have a bootstrap skin in the works for TinyMCE 5 and it would be cool if you'd like to try it out. If so, get in touch at [fredrik.danielsson@tiny.cloud](mailto:fredrik.danielsson@tiny.cloud). Cheers!

That would be great and then I can try to get my filemanager to integrate

**Comment by TheSpyder** [comment]
Temporarily un-pinning this as we are limited to 3 pins

**Comment by androb** [comment]
Bringing this back. We'd love to hear what you have built with TinyMCE!

**Comment by cphill02** [comment]
4.7.9 checking in...
Currently JQuery based custom framework migrating to React w/ Bootstrap. _Node.js on the back side._
![TinyMCE](https://user-images.githubusercontent.com/1103324/58607219-5a030a80-8264-11e9-9745-b14e17d1576b.png)

**Comment by lostkeys** [comment]
C'mon add your TinyMCE implementations here! We want to see them all!

**Comment by Anakowi** [comment]
Oh yeah, forgot I was going to do that. Here'tis - https://www.popupweb.com.au/penpages/
Note DEMO link (top right) — pm for user/pw, thanks.

**Comment by MirzayevFarid** [comment]
- We’re using TinyMCE at [Radity](https://radity.com/)

- ![logo-midnight@3x](https://user-images.githubusercontent.com/29518613/91814291-f32d8780-ec44-11ea-98ae-a99de71fbdc6.png)

**Comment by tolunayakbulut** [comment]
[Global CTO Forum ](https://globalctoforum.org/about-us/) is using TinyMCE.

![global-cto-forum](https://user-images.githubusercontent.com/31761883/101292711-44d84600-3822-11eb-8cd4-e68310ca54e4.png)

**Comment by lbreza** [comment]
![logo_2_complete](https://user-images.githubusercontent.com/44499223/104020443-86f7fd00-51bd-11eb-806c-6ed6e8868625.png)

[Organisify](https://organisify.com/) is using TinyMCE (latest version).

![aaa](https://user-images.githubusercontent.com/44499223/103982331-96f2eb00-5183-11eb-9837-447fe8a45eae.png)

**Comment by Anakowi** [comment]
[Penpages](https://www.popupweb.com.au/penpages/) is using TinyMCE 
![ppMiniBlack-Logo](https://user-images.githubusercontent.com/46861039/104087320-16cba300-52b3-11eb-81c4-4f405d74f990.png)
 :-)

**Comment by Philipp91** [comment]
I believe the [Joplin](https://joplinapp.org/) note app [is using](https://github.com/laurent22/joplin/blob/29a8e815d92c3f3cf8faef9056203085d713ceee/packages/app-desktop/package.json#L162) TinyMCE, I guess @laurent22 can tell you more.

**Comment by laurent22** [comment]
Indeed we use it on the desktop application for the rich text editor. 

* Company name: joplinapp.org
* Logo: https://raw.githubusercontent.com/laurent22/joplin/dev/Assets/LinuxIcons/256x256.png
* Type of project: note taking application
* It's an open source project

![Joplin Logo](https://raw.githubusercontent.com/laurent22/joplin/dev/Assets/LinuxIcons/128x128.png)

**Comment by vmato** [comment]
Still using but suffering from issues like #3564

**Comment by NicolasCARPi** [comment]
Oh yes we use it and love it!

[Deltablot](https://www.deltablot.com/): Open Source software for research labs
[![deltablot-logo](https://user-images.githubusercontent.com/3043706/120047180-47b73500-c014-11eb-84b4-c7d7ad236129.png)](https://www.deltablot.com)

Project: free and open source electronic lab notebook: :arrow_right:  [GitHub repo](https://github.com/elabftw/elabftw/) :arrow_left: 
Online demo: https://demo.elabftw.net

![2021-05-29-002703_1466x666_scrot](https://user-images.githubusercontent.com/3043706/120047339-9c5ab000-c014-11eb-82c3-1eb2dd6c7731.png)

**Comment by aceArt-GmbH** [comment]
![aceart](https://user-images.githubusercontent.com/33117017/136522274-2bb84084-4ea3-4ef3-bc98-56c709b66fef.png)
https://www.aceart.de/

We use TinyMCE as part of an custom ERP system.
Small team but big customer
